tests/test_cli.py

```diff
@@ -134,23 +131,17 @@ def test_end_to_end_armormaxsize(capsys, tmp_path):
     f.write(b"\0")
 
   # Encrypt test.dat with armor and no padding
-  sys.argv = f"covert -eR tests/keys/ssh_ed25519.pub --pad 0 -ao".split() + [ str(outfname), str(fname) ]
-  ret = main()
-  cap = capsys.readouterr()
-  assert not ret
+  cap = covert("enc", fname, "-R", "tests/keys/ssh_ed25519.pub", "--pad", 0, "-ao", outfname)
```

Comment:
Why don't we use short arguments combined?
```suggestion
  cap = covert("-eR", "tests/keys/ssh_ed25519.pub", "--pad", 0, "-ao", outfname, fname)
```

Reply:
I am trying to have some variation in the tests, doing things differently in each test to try to break more things (and run more code). `fname` was moved from the end to the beginning, preventing the use of the original short arg.